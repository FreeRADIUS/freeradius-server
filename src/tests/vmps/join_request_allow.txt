#
#	ARGV: -v tapioca.lan -w 192.168.69.1 -i eth0 -m cafe.cafe.cafe -c Kalos0
#
