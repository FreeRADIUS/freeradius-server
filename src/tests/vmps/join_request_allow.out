Vlan: Kalos0
MAC Address: cafecafecafe 
Status: ALLOW
